packages/jellyfish-api-core/__tests__/category/masternode/getActiveMasternodeCount.test.ts

```diff
@@ -0,0 +1,27 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+
+describe('Masternode', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should getActiveMasternodeCount', async () => {
+    jest.setTimeout(400000)
+    await client.masternode.createMasternode(await client.wallet.getNewAddress())
+
+    await container.generate(2016)
```

Comment:
I think you need to generate to the created masternode as well.

I think this is impossible to test without a multi-node setup, let's wait on this.

blocked by https://github.com/DeFiCh/jellyfish/issues/40

Reply:
@fuxingloh ,  I can now enable a masternode via `container.generate(nBlocks, address)` but i am yet to make it active. 

The active masternode count  does not change even after enabling a masternode. 

I thought enable means active but i guess i am wrong 

my question here is: What does active mean in regards to the `getActiveMasternode` RPC? 


```javascript
   await client.masternode.getActiveMasternodeCount() // 1

    const ownerAddress = await container.getNewAddress()

    await client.masternode.createMasternode(ownerAddress)
    await container.generate(20, ownerAddress) // Enables masternode

    await client.masternode.getActiveMasternodeCount() // 1
```

Reply:
Need to wait for https://github.com/DeFiCh/jellyfish/pull/470 to be merged

Sorry I am a little too occupied now to review that

Reply:
Okay.